LEINS-limited/Kabstore-mis-backend
# feat(auth): added mailing service, deployment and auth services

## Description
 This pull request introduces more auth options like account verifications and reset, plus mailing and render deployment configurations.

### Changes
**Deployment**: Deployed the app on render.
**Mailing service:**  Replaced NodeMailer with MailingService from Nest.
**Enhanced auth functionalities:** Introduced more auth options like account verification and reset.

### How to Test?

- Run the app.
- Execute APIs.
- Verify that all endpoints work as expected without any errors or failures.

CC @U-Alice
